src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.fr.xlf

```diff
@@ -74,7 +74,7 @@
       </trans-unit>
       <trans-unit id="LocalizedControlTypeHyperlink">
         <source>hyperlink</source>
-        <target state="translated">lien hypertexte</target>
+        <target state="translated">hyperlink</target>
```

Comment:
All LocalizedControlType* strings should remain localized for all languages.  Please review LocalizedControlTypeTable as well.